source/routing/tests/raptor_test.cpp

```diff
@@ -2044,7 +2046,7 @@ BOOST_AUTO_TEST_CASE(direct_path_filter) {
     // reverse clockwise
     res = raptor.compute_all(departures,
                              arrivals,
-                             DateTimeUtils::set(2, "08:03"_t),
+                             DateTimeUtils::set(2, "08:05"_t),
```

Comment:
The test was buggy: we were getting the journey of the day before. Now that there is a 24h hard limit (else we have memory problems...), the test can't find the journey of the day before.